_assets/javascripts/_includes/panels/action_w_call_congress.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function(){
+var views = $('.js-panel-action-w-call-congress');
+views.each( function(i, el){
```

Comment:
@Rio517 Because we want to avoid using backbone, and we need to make sure each panel is hit by the view logic individually, I think this is the best way to do it. This requires using wrapper classes for every panel, as well as only searching for elements on the 'el' in the view. Mustache does not seem to have a way to achieve this level of individual logic. 

I think this is really a question of how much complexity you can foresee on the front end. If there are multiple instances of duplicate views being created and destroyed, Backbone (or an equivalent DOM logic management system like react or angular) is going to quickly become worthwhile, but if you don't think thats going to happen soon, we can just leave this as is. 

Also if you can think of a better way to set this up, please do let me know.